NBug/NBug.csproj

```diff
@@ -318,23 +308,5 @@
       <DependentUpon>Normal.cs</DependentUpon>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PostBuildEvent Condition="$(OS) == 'WINDOWS_NT'">if $(ConfigurationName)==Release (

```

Comment:
⚠️ I couldn't figure out why this was used, but part of my work was avoiding the use of `PreBuildEvent` and `PostBuildEvent` since they are prone to failing on incremental builds.